backend/index.js

```diff
@@ -41,6 +40,7 @@ app.all('*', function (req, res, next) {
     app.get('/photo/turtle/:turtleId', db.getPhotoByTurtleId)
     app.get('/photo/sighting/:sightingId', db.getPhotoBySightingId)
     app.get('/email/:address', db.sendEmail)
+    next() // Go onto the queries.
```

Comment:
This fixes an issue where the first request always fails. I added it to this pull request just to make it simpler to review.

Reply:
Thanks! That bug is so annoying. Why do you think the first one always fails?

Reply:
I think that when the first request came, next() would be called before the routes were defined, so it wouldn't be routed but the routes would still get defined after a response was sent. Then, on the next call, the routes would have already been defined from the previous call so there wouldn't be an issue.

Reply:
Ah makes sense. Thanks!